win32ss/gdi/ntgdi/text.c

```diff
@@ -3,7 +3,7 @@
  * LICENSE:         GPL - See COPYING in the top level directory
  * FILE:            win32ss/gdi/ntgdi/text.c
  * PURPOSE:         Text/Font
- * PROGRAMMER:
+ * PROGRAMMER:      Katayama Hirofumi MZ
```

Comment:
I'd recommend looking at the history and adding the other authors as well, this looks odd otherwise.

Reply:
I agree.

Reply:
https://git.reactos.org/?p=reactos-deprecated-gitsvn-dont-use.git;a=history;f=reactos/win32ss/gdi/ntgdi/text.c;h=b4130b6b2fc0423a13824e794122179ca85492cd;hb=refs/heads/master
The other authors were akhaldi, hbelusca, jimtabor, and tkreuzer.

Reply:
OK, I added the authors.